plugins/scoreboard.js

```diff
@@ -86,41 +87,57 @@ function getEntitiesCount (entities,bounds) {
   return {
     enl: counts[window.TEAM_ENL] || 0,
     res: counts[window.TEAM_RES] || 0,
+    mac: counts[window.TEAM_MAC] || 0,
   };
 }
 
 function makeTable (portals,linksCount,fieldsCount) {
 
   var html = '';
   html += '<table>'
-  + '<colgroup><col><col class="enl"><col class="res"></colgroup>'
+  + '<colgroup><col><col class="enl"><col class="res"><col class="mac"></colgroup>'
   + '<tr>'
   + '<th>Metrics</th>'
   + '<th class="enl">Enlightened</th>'
   + '<th class="res">Resistance</th>'
```

Comment:
These names, and css classes used above are saved in constants similar to e.g. TEAM_MAC
Please use those instead of repeating the value